### PR TITLE
Fix rubocop issues

### PR DESCRIPTION
**Pull request by phyrog**
Fixes all present rubocop issues or disables checks for exceptions.

### CODE REVIEW

File: config/initializers/settings_validator.rb [hunk 1]
```diff
@@ -58,7 +58,11 @@ def print_errors
   # :nocov:
 
   # Checker methods
+  # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/MethodLength, Metrics/PerceivedComplexity
   def check_server_url
```
Comment (phyrog) [inline config/initializers/settings_validator.rb]: This method might be worth to refactor (given how many rules it violates), however I didn't have a good alternative, so I just disabled the failing checks for now.

File: .rubocop_disabled.yml [hunk 2]
```diff
@@ -35,30 +64,6 @@ Style/Encoding:
...
-                 Checks for a line break before the first argument in a
-                 multi-line method call.
-  Enabled: false
-
-Layout/FirstMethodParameterLineBreak:
-  Description: >-
-                 Checks for a line break before the first parameter in a
-                 multi-line method parameter definition.
-  Enabled: false
-
```
Comment (phyrog) [inline .rubocop_disabled.yml]: I just downloaded the new version of the default config from the rubocop repo at the `0.49.1` tag.

Comment (eugenk) [inline .rubocop_disabled.yml]: I see

**Comment by eugenk** [comment]
Could you please also add rubocop to the Gemfile?

File: .rubocop_disabled.yml (hunk 2 above)
Comment (phyrog) [inline .rubocop_disabled.yml]: I tried the version from the master branch, but they seem to have added some cops, so they will throw an error (or a warning) on 0.49.1.

**Comment by phyrog** [comment]
Done in 30afe0c

**Comment by phyrog** [comment]
Well... I guess I found the hound problem

**Comment by phyrog** [comment]
Rebased on master

**Pull request merged**